skl2onnx/operator_converters/bagging.py

```diff
@@ -0,0 +1,133 @@
+# --------------------------------------------------------------------------
+
+import numpy as np
+from ..common._apply_operation import (
+    apply_cast, apply_concat,
+    apply_div, apply_reshape,
+)
+from ..common._registration import register_converter
+from ..common._topology import FloatTensorType
+from ..proto import onnx_proto
+from .._supported_operators import sklearn_operator_name_map
+
+
+def _calculate_proba(scope, operator, container, model):
+    """
+    This function calculates class probability scores for
+    BaggingClassifier.
+    """
+    final_proba_name = operator.outputs[1].full_name
+    proba_list = []
+    has_proba = hasattr(model.estimators_[0], 'predict_proba')
+    for index, estimator in enumerate(model.estimators_):
+        op_type = sklearn_operator_name_map[type(estimator)]
+
+        this_operator = scope.declare_local_operator(op_type)
+        this_operator.raw_operator = estimator
+        this_operator.inputs = operator.inputs
+
+        label_name = scope.declare_local_variable('label_%d' % index)
+        proba_name = scope.declare_local_variable('proba_%d' % index,
+                                                  FloatTensorType())
+        this_operator.outputs.append(label_name)
+        this_operator.outputs.append(proba_name)
+        proba_output_name = (proba_name.onnx_name if has_proba
+                             else label_name.onnx_name)
+        reshape_dim_val = len(model.classes_) if has_proba else 1
+        reshaped_proba_name = scope.get_unique_variable_name('reshaped_proba')
+        apply_reshape(scope, proba_output_name, reshaped_proba_name,
+                      container, desired_shape=(1, -1, reshape_dim_val))
+        proba_list.append(reshaped_proba_name)
+    merged_proba_name = scope.get_unique_variable_name('merged_proba')
+    apply_concat(scope, proba_list,
+                 merged_proba_name, container, axis=0)
+    if has_proba:
+        container.add_node('ReduceMean', merged_proba_name,
+                           final_proba_name,
+                           name=scope.get_unique_operator_name('ReduceMean'),
+                           axes=[0], keepdims=0)
+    else:
+        n_estimators_name = scope.get_unique_variable_name('n_estimators')
+        class_labels_name = scope.get_unique_variable_name('class_labels')
+        equal_result_name = scope.get_unique_variable_name('equal_result')
+        cast_output_name = scope.get_unique_variable_name('cast_output')
+        reduced_proba_name = scope.get_unique_variable_name('reduced_proba')
+
+        container.add_initializer(
+            n_estimators_name, onnx_proto.TensorProto.FLOAT, [],
+            [len(model.estimators_)])
+        container.add_initializer(
+            class_labels_name, onnx_proto.TensorProto.INT64,
+            [1, 1, len(model.estimators_[0].classes_)],
+            model.estimators_[0].classes_)
+
+        container.add_node('Equal', [class_labels_name, merged_proba_name],
```

Comment:
Should we add these two op into onnx_ops.apply_*, given they are popular?

Reply:
If there are multiple versions of the op, we may think of adding it.